index.html

```diff
@@ -3187,15 +3189,22 @@ <h3 id="h-sourcebufferlist-events" resource="#h-sourcebufferlist-events"><span p
         </thead>
         <tbody>
           <tr>
-            <td>06 June 2016</td>
+            <td>07 June 2016</td>
+            <td>
+              <ul>
+                <li>Issue 69 - Add optional &amp; subject to change language to <a href="#videoplaybackquality">VideoPlaybackQuality</a>.</li>
```

Comment:
rebase to current gh-pages, make sure publishDate is correct, and regenerate index.html to retain change log for things that have landed in the interim. Otherwise, LGTM.